lua/kulala/parser/init.lua

```diff
@@ -693,6 +693,7 @@ M.parse = function(start_request_linenr)
   table.insert(res.cmd, "@" .. CURL_FORMAT_FILE)
   table.insert(res.cmd, "-X")
   table.insert(res.cmd, res.method)
+  table.insert(res.cmd, "-v")
```

Comment:
Haven't tested it yet, but doesn't this cause issues for non-verbose requests?

Reply:
I have not noticed any.  The only difference for verbose requests is that curl pipes verbose output to stderr.  Non verbose  requests just ignore it.